build-infrastructure/README.md

```diff
@@ -0,0 +1,56 @@
+# Building the ECS Agent using AWS CodeSuite
+
+## Prerequisites
+
+There are some things you need to get this going
+
+1. Your favorite AWS region
+1. A GPG key stored in AWS Secrets Manager
```

Comment:
Should these be increasing numbers? or bullets points?

Reply:
Markdown automatically increases the numbers, to denote an ordered list, you only need to satisfy the regex `/[0-9]\./`